velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -230,7 +236,7 @@ std::string toCallSql(
     } else if (
         auto call =
             std::dynamic_pointer_cast<const core::CallTypedExpr>(input)) {
-      sql << toCallSql(call, {}, {});
+      sql << toCallSql(call, {}, {}, false);
```

Comment:
nit: for readability, perhaps, split toCallSql into 2 functions: toAggregateCallSql for aggregate functions (with 4 args) and toCallSql for other functions (with 1 arg)

Reply:
Got it, done.